ceph/ceph_admin/cephadm_ansible.py

```diff
@@ -39,7 +41,8 @@ def __init__(self, cluster):
 
     def install_cephadm_ansible(self):
         """Enable ansible rpm repos and install cephadm-ansible."""
-        if float(self.cluster._Ceph__rhcs_version) >= 6.0:
+        rhcs_version = re.findall(r"(\d.\d)(.*)", self.cluster._Ceph__rhcs_version)
+        if rhcs_version and float(rhcs_version) >= 6.0:
```

Comment:
Hi Vaibhav,

As per the recent failure 

"""
 if float(self.cluster._Ceph__rhcs_version) >= 6.0:
ValueError: could not convert string to float: '5.3-rhel-8'

"""
so the value of rhcs_version  is `5.3-rhel-8` right? the re.findall on this value returns => ` [('5.3', '-rhel-8')]`

When trying the above logic,

"""
ceph__rhcs_version = "5.3-rhel-8"
rhcs_version = re.findall(r"(\d.\d)(.*)", ceph__rhcs_version)
print(rhcs_version)
if rhcs_version and float(rhcs_version) >= 6.0:
    print("RHCEPH 6")"""
    
    
I can see the below error
"""
TypeError: float() argument must be a string or a number, not 'list'
"""

Reply:
@pranavprakash20 good catch, missed to call group function